script.js

```diff
@@ -741,6 +741,37 @@ let projectData = [
     projectUrl: "public/MathQuiz.html"
   },
   {
+
```

Comment:
Lot of projects are added again, It will create a duplicacy. Only add your project